client/src/reducer.js

```diff
@@ -9,19 +12,28 @@ const reducer = (state, action) => {
     case 'DISPLAY_SEARCH_RESULTS':
       return { ...state, searchResults: action.payload, searchError: null };
 
-    case 'NO_SEARCH_RESULTS':
+    case 'NO_SEARCH_RESULTS': {
+      const message =
+        state.currentLanguage === 'ro'
+          ? 'Nu a fost găsită nici o clădire cu adresa'
+          : 'No building was found matching the address';
```

Comment:
I'd like these kinds of messages to be implemented with `switch` statements instead of `if` statements. The reason is that we will have to extend the internationalisation to (at least) 3 languages and 🇭🇺  will also make the list soon